client/src/pages/Pictures.js

```diff
@@ -10,22 +10,22 @@ const PicturesDiv = styled.div`
   display: flex;
   flex-direction: column;
   align-content: center;
+  max-width: 600px;
 `;
 
 const Pictures = () => {
   return (
     <PicturesDiv>
+      <PageHeadline>Bildname</PageHeadline>
       <PictureContainer>
         <img src={placeholder} alt="placeholder" />
       </PictureContainer>
-      <PageHeadline>Suche</PageHeadline>
       <p>
-        Der Dom. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
-        diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam
-        erat, sed diam voluptua.{" "}
+        Bildbeschreibung Lorem ipsum dolor sit amet, consetetur sadipscing
```

Comment:
Don't forget to remove the `Lorem ipsum` text in the end :)

Reply:
At the time its just a placeholder as a want to read the data from a database wich is not yet set up :)